internal/errors/worker.go

```diff
@@ -43,4 +43,9 @@ var (
 	ErrJobFuncIsNil = func() error {
 		return New("JobFunc is nil")
 	}
+
+	// ErrJobFuncIsNil represents a function to generate job function is not found.
+	ErrJobFuncNotFound = func() error {
+		return New("JobFunc is not found")
+	}
```

Comment:
It is used for https://github.com/vdaas/vald/pull/1879/files#diff-2666dcdb36edfd61cbb3e1af4d90cbe4c038636caf249f809fc152a3cfc4d0efR154